.project

```diff
@@ -14,4 +14,15 @@
 	<natures>
 		<nature>org.eclipse.m2e.core.maven2Nature</nature>
 	</natures>
+	<filteredResources>
```

Comment:
why did you add this? was this on purpose?

Reply:
no, it wasn't on purpose. I'll remove it, thanks